src/main/io/displayport_msp_bf_compat.c

```diff
@@ -65,32 +65,33 @@ uint8_t getBfCharacter(uint8_t ch, uint8_t page)
 /*
     case SYM_HEADING:
         return BF_SYM_HEADING;
+*/
 
     case SYM_SCALE:
-        return BF_SYM_SCALE;
+        return 'I';
 
     case SYM_HDP_L:
-        return BF_SYM_HDP_L;
+        return BF_SYM_RSSI;
 
     case SYM_HDP_R:
-        return BF_SYM_HDP_R;
-*/
+        return BF_SYM_RSSI;
+
     case SYM_HOME:
         return BF_SYM_HOMEFLAG;
 
     case SYM_2RSS:
         return BF_SYM_RSSI;
 
-/*
     case SYM_DB:
-        return BF_SYM_DB
+        return 'd' + 'B';
 
     case SYM_DBM:
-        return BF_SYM_DBM;
+        return 'd' + 'B' + 'm';
```

Comment:
You can't do this. Symbols are a single space character. Adding multiple character will cause issues with the OSD creation.